python/ray/util/collective/util.py

```diff
@@ -40,3 +40,29 @@ def get_id(self):
             logger.warning("The NCCL ID has not been "
                            "set yet for store {}.".format(self.name))
         return self.nccl_id
+
+
+@ray.remote
+class Info:
```

Comment:
Can you please rename? Also, this is just a makeshift kvstore right?

Can we maybe use `ray.experimental.internal_kv` instead of this actor?

Reply:
@zhisbug let's chat offline about possible alternatives here? this may seem harmless but I think it could easily be a source of issues later on.

Reply:
let's talk tomorrow. If the `internal_kv` is good we can refactor all named_actor code to use this one (which might be better!)